Dataform/src/V1beta1/CreateRepositoryRequest.php

```diff
@@ -16,8 +16,8 @@
 class CreateRepositoryRequest extends \Google\Protobuf\Internal\Message
 {
     /**
-     * Required. The location in which to create the repository. Must be in the format
-     * `projects/&#42;&#47;locations/&#42;`.
+     * Required. The location in which to create the repository. Must be in the
+     * format `projects/&#42;&#47;locations/&#42;`.
```

Comment:
The corresponding line is correctly formatted, [ref](https://source.corp.google.com/piper///depot/google3/third_party/googleapis/stable/google/cloud/dataform/v1beta1/dataform.proto;l=684;rcl=576787262).

Reply:
This is an issue with protobuf, which I made a fix for here: https://github.com/protocolbuffers/protobuf/pull/11208

In the meantime, there's nothing we can do about it (and it's okay to merge as such, since a documentation formatting issue shouldn't block releasing updates)